### PR TITLE
Update opentracing doc to describe using application.properties as alternative to env vars

### PR DESCRIPTION
**Pull request by objectiser**
This PR accompanies quarkusio/quarkus-quickstarts#107.

### CODE REVIEW

File: docs/src/main/asciidoc/opentracing-guide.adoc [hunk 1]
```diff
@@ -82,16 +102,19 @@ The first step is to start the tracing system to collect and display the capture
 docker run -e COLLECTOR_ZIPKIN_HTTP_PORT=9411 -p 5775:5775/udp -p 6831:6831/udp -p 6832:6832/udp -p 5778:5778 -p 16686:16686 -p 14268:14268 -p 9411:9411 jaegertracing/all-in-one:latest
 ----
 
-Now we are ready to run our application. Use:
+Now we are ready to run our application. If using `application.properties` to configure the tracer:
 
 [source, text]
 ----
-./mvnw compile quarkus:dev -Djvm.args="-DJAEGER_SERVICE_NAME=myservice -DJAEGER_SAMPLER_TYPE=const -DJAEGER_SAMPLER_PARAM=1 -DJAEGER_ENDPOINT=http://localhost:14268/api/traces"
+mvn compile quarkus:dev
```
Comment (gsmet) [inline docs/src/main/asciidoc/opentracing-guide.adoc]: We decided to go with the Maven wrapper approach as we had a few reports with people not using the right version of Maven. Could you change it back to `./mvnw`?

File: docs/src/main/asciidoc/opentracing-guide.adoc [hunk 2]
```diff
@@ -82,16 +102,19 @@ The first step is to start the tracing system to collect and display the capture
...
 ----
 
-The supplied `jvm.args` are used to configure the default OpenTracing tracer (Jaeger). If the `JAEGER_SERVICE_NAME` property is not provided then a "no-op" tracer will be configured, resulting in no tracing data being reported to the backend. The other properties are explained in the https://www.jaegertracing.io/docs/latest/client-features/[Tracer configuration via environment variables] section of the Jaeger documentation.
+or if configuring the tracer via environment variables:
 
-NOTE: Currently the tracer can only be configured to report spans directly to the collector via HTTP, using the `JAEGER_ENDPOINT` property. Support for using the Jaeger agent, via UDP, will be available in a future version.
+[source, text]
+----
+mvn compile quarkus:dev -Djvm.args="-DJAEGER_SERVICE_NAME=myservice -DJAEGER_SAMPLER_TYPE=const -DJAEGER_SAMPLER_PARAM=1 -DJAEGER_ENDPOINT=http://localhost:14268/api/traces"
+----
```
Comment (gsmet) [inline docs/src/main/asciidoc/opentracing-guide.adoc]: Same here.

**Review by gsmet — CHANGES_REQUESTED** [review]
@objectiser looks good to me, thanks.

I added a minor comment about the usage of `./mvw`. Could you fix that and I'll merge? Thanks!

**Comment by objectiser** [comment]
@gsmet updated. Thanks.

**Review by gsmet — APPROVED** [review]

**Comment by gsmet** [comment]
Merged, thanks!

**Pull request merged**